proto/config/netinst.proto

```diff
@@ -153,4 +153,18 @@ message NetworkInstanceConfig {
   // This option is only valid for local network instances. For other types
   // of network instances, it will be ignored.
   repeated IPRoute static_routes = 43;
+
+  // Maximum transmission unit (MTU) to set for the network instance and all application
+  // interfaces connected to it.
+  // This is the largest IP packet that the network instance is allowed to carry.
+  // The value is a 16-byte unsigned integer, representing the MTU size in bytes.
+  // The minimum legal value for the MTU is 68 (RFC 791).
+  // If not defined (zero value), EVE will set the MTU to the default value of 1500 bytes.
+  // On the host side, MTU is set to interfaces by EVE. On the guest (application) side,
+  // it is the role of the DHCP client to apply the MTU value, which it will receive with
+  // the DHCP lease via option 26.
+  // Please note that application traffic leaving or entering the device via a network
+  // adapter associated with the network instance will be additionally limited by the MTU
+  // value of the adapter, configured inside the NetworkConfig object (see netconfig.proto).
```

Comment:
OK, makes sense, added comment about this.

Reply:
However, I'm not sure how to handle the switch network instance. In that case, the MTU for the network adapter and the MTU for the switch NI reference the same bridge. I believe we should then flag both the adapter and NI with an error if the MTUs are not the same (and use the lower of the two values for the time being). WDYT?

Reply:
Makes sense.

But what about a local network instance? Will the NAT fragment automatically if the external ports and internal bridge MTU differs? Or do we need to also check those for equality?

Reply:
I decided that we will be forcing MTU equality between NI and the associated adapter for both switch and local NIs. Otherwise we may have undesired IP fragmentation happening inside EVE.